Add a bunch of apps

MiepHD opened the pull request:
Antimine, Birday, Joplin, KeePass2Android, KeePassDroid, MicroG, Organic Maps, SAI, Session, SmartTube

MiepHD committed:
Added FairEmail, IronFox, KeePass2Android, Organic Maps, SAI, SMS Ping

MiepHD committed:
Added MicroG for ReVanced Apps, MicroG, ReVanced Manager, ReVanced Youtube Music, RVX YT Extended, RVX YT Music Extended, Youtube ReVanced

MiepHD committed:
Added SmartTube, Adguard, Aliucord Manager, Antimine, Birday, Brave - Beta, ByCS Messenger, KeePassDroid, Session

MiepHD committed:
Removed ReVanced links

MiepHD commented:
I didn't know they were unofficial

MiepHD commented:
All apps that have apk releases on github already link to github

Oriya-osmine commented on repo/apps.yml:
```diff
@@ -581,6 +581,28 @@
+  id: app.revanced.android.gms
+  type: submitted
+  verified: false
+  releases:
```
this shouldn't be the release url, this should be for example https://github.com/ReVanced/GmsCore/releases
wrong lines, the line under what that is shown

Oriya-osmine commented:
> Should I remove the others?

what others?

Oriya-osmine commented:
> As far as I know there are no offical packaged apk files ready for download for revanced apps

not for patched apps, but for the basic ones such as [revanced manager](https://github.com/revanced/revanced-manager)

MiepHD commented:
ReVanced Manager is already in Discoverium

Oriya-osmine commented on repo/apps.yml:
```diff
@@ -9,6 +9,17 @@
+  type: submitted
+  verified: true
+  releases:
+    url: https://download.adguardcdn.com/d/18673/adguard.apk
```
`url: https://download.adguardcdn.com/d/18673/adguard.apk`?

Oriya-osmine commented on repo/apps.yml:
```diff
@@ -42,6 +64,17 @@
+  type: submitted
+  verified: true
+  releases:
+    url: https://f-droid.org/packages/dev.lucanlm.antimine/
```
link this instead https://github.com/lucasnlm/antimine-android/releases

MiepHD committed:
Changed type to submitted and removed apps that don't link to github

Oriya-osmine reviewed (approved):
good job

MiepHD commented:
Brave contains advertisements and FairEmail has Premium Features

MiepHD commented:
I would really appreciate it if you could manage to add IronFox too. (It's just not on Github but Gitlab)

Oriya-osmine commented:
> I would really appreciate it if you could manage to add IronFox too. (It's just not on Github but Gitlab)
Gitlab is supported, but yes it should be refined further to allow multiple release urls or something like that

Oriya-osmine commented:
> > > I would really appreciate it if you could manage to add IronFox too. (It's just not on Github but Gitlab)
> > 
> > Gitlab is supported, but yes it should be refined further to allow multiple release urls or something like that
> > 
> > 
> Oh, ok. Then I'll put IronFox back into the PR
> 
You can see the readme for what sources are supported

MiepHD commented:
So I can put Antimine (F-Droid Link) back in too?

Oriya-osmine commented:
> So I can put Antimine (F-Droid Link) back in too?

Yep, let's continue in the discord(press on the discord icon in readme) if you have questions, I don't want to clutter this

MiepHD committed:
Added Antimine, IronFox, Joplin, Seal, Shadowsocks
Removed Brave, FairEmail

MiepHD committed:
Removed SMS Ping and fixed Aliucord Manager URL

MiepHD committed:
Changed authors of IronFox and MicroG

MiepHD committed:
Added author for Aliucord Manager
Added full name to SmartTube
Removed Shadowsocks, Seal, SAI as I can't find any real author

MiepHD committed:
Removed Aliucord Manager, IronFox
Added real names for Birday, KeePass2Android, KeePassDroid, MicroG

MiepHD committed:
Added author for SAI

MiepHD committed:
Added real name for Cuscon

MiepHD committed:
Extended name

MiepHD committed:
Corrected Organic Maps name

MiepHD committed:
Remove Cuscon as it does not compy with the Admission policy

MarkyMarkDE commented:
am I'm allowed to put the facts of my Apps / Devs here too (Github related i mean)?

MarkyMarkDE commented:
> I think it's better if you treat this one as closed and open your own one

prefered headline for this?
